_posts/2022-11-19-sr2023-kickstart-and-rules.md

```diff
@@ -0,0 +1,25 @@
+---
+title: SR2023 Game Announced
+---
+
+Student Robotics 2023 was Kickstarted on Saturday 22nd September at the University of Southampton and on our [livestream][livestream]. It was great to see the excitement building at our first in-person Kickstart since SR2019.
+
+Our game this year, _Greed_, challenges teams to steal tokens from other scoring zones. With Bronze, Silver, and Gold tokens, each worth different amount of game points, teams must think carefully about the optimum strategy. To help teams locate tokens they have 2D barcode style markers attached that our computer vision library can detect. However each of the markers will identify themselves as exactly the same, so teams must use other sensors to detect which type of token is which. Bronze and Silver tokens are the same size but the Silver tokens weigh 300g more, and Gold Tokens are larger than Bronze and Silver. Teams are scored at the end of the match for collecting the most points. Full details, including the prizes available this year, are available in the [rulebook][rules].
```

Comment:
There's one case of capitalised Tokens here ("Gold Tokens are larger") yet the rest of the mentions are lower case. Suggest making them consistent.